lib/shopify_api/resources/line_item.rb

```diff
@@ -2,11 +2,11 @@ module ShopifyAPI
   class LineItem < Base
     class Property < Base
       def initialize(*args)
-        attributes = args[0] || {}
-        persisted = args[1] || false
+        self.attributes = args[0] || {}
+        self.persisted = args[1] || false
         super
       rescue NameError
-        attributes = attributes.to_hash
+        self.attributes = attributes.to_hash
```

Comment:
I wasn't sure whether to do this or ignore this particular issue, but it doesn't seem like these variables were used at all in the previous code.

Reply:
Yeah, it seems flawed.